docs/src/main/asciidoc/virtual-threads.adoc

```diff
@@ -80,7 +80,12 @@ It will either reduce the chance for the other virtual thread to run or will sta
 
 In Quarkus, the support of virtual thread is implemented using the link:{runonvthread}[@RunOnVirtualThread] annotation.
 This section briefly overviews the rationale and how to use it.
-There are dedicated guides for extensions supporting that annotation, such as // TODO.
+There are dedicated guides for extensions supporting that annotation, such as:
```

Comment:
```suggestion
There are dedicated guides for extensions supporting that annotation:
```

"Such as" suggests that what follows is just examples and there may be more, but you provide a complete list here IIUC?

Reply:
The list is complete right now, but it will be extended, potentially in the quarkiverse. That's why I wanted to phrase it as a non exhaustive list.

Reply:
Fair enough.